snap/info.go

```diff
@@ -385,6 +386,8 @@ type DownloadInfo struct {
 	Size     int64  `json:"size,omitempty"`
 	Sha3_384 string `json:"sha3-384,omitempty"`
 
+	Updated time.Time `json:"updated"`
```

Comment:
it's a bit strange to put it here, is it to follow Size? but the deduced value for size is the same for local and remote, that is not true for Updated